src/main/java/com/baidu/hugegraph/structure/constant/ReadFrequency.java

```diff
@@ -23,7 +23,11 @@
 
     OLTP(1, "oltp"),
 
-    OLAP(2, "olap");
+    OLAP_NONE(2, "olap_none"),
+
+    OLAP_SECONDARY(3, "olap_secondary"),
+
+    OLAP_RANGE(4, "olap_range");
```

Comment:
not sure when it need to pass OLAP_SECONDARY or OLAP_RANGE to server

Reply:
add comments